frame/support/src/crypto/ecdsa.rs

```diff
@@ -41,7 +41,7 @@ impl ECDSAExt for Public {
 			let uncompressed = pub_key.to_encoded_point(false);
 			// convert to ETH address
 			<[u8; 20]>::try_from(
-				sp_io::hashing::keccak_256(&uncompressed.as_bytes()[1..])[12..].as_ref(),
+				sp_io::hashing::keccak_256(uncompressed.as_bytes())[12..].as_ref(),
```

Comment:
The (standard) byte-encoded form for uncompressed pub keys still uses the first byte to express that the byte sequence represents the uncompressed form (i.e. 0x04 value for the tag).
(https://github.com/RustCrypto/formats/blob/bbab401934a2e93d28053fd74613ffef74cd6e88/sec1/src/point.rs#L132)